indykite/utilities.go

```diff
@@ -409,6 +409,13 @@ var OAuth2ClientSubjectTypes = map[string]configpb.ClientSubjectType{
 // OAuth2ClientSubjectTypesReverse defines all supported Client Subjects and its reversed mapping.
 var OAuth2ClientSubjectTypesReverse = ReverseProtoEnumMap(OAuth2ClientSubjectTypes)
 
+// AuthorizationPolicyStatusTypes defines all supported StatusTypes and its mapping.
+var AuthorizationPolicyStatusTypes = map[string]configpb.AuthorizationPolicyConfig_Status{
+	"invalid":  configpb.AuthorizationPolicyConfig_STATUS_INVALID,
```

Comment:
Remove `invalid`. It is invalid anyway, if that value is received, we should log it.